### DISCUSSION TITLE
Range searches on iterations

### DISCUSSION THREAD

**Discussion by bertptrs** [community/community#14368] [Projects and Issues]
On the project boards we can filter on a specific iteration, either with an exact match or relative to the current time (`@current` etc). However, for many queries I find myself having the need to express things like "before the current iteration", "before some specific date", and "in future iterations".

I tried using the inequality operators for this but it seems that does not work (yet).

**Comment by CalembourIII** [comment]
Hi, I found that it kinda works if you play with the filters like this: 

iteration:<@next+1 -iteration:<=@current-12

Although you have to maintain the filter if you use it constantly. But it works :)
